cdap-docs/reference-manual/source/http-restful-api/metadata.rst

```diff
@@ -1302,3 +1333,55 @@ with the metadata returned as a JSON string in the return body::
      - The properties requested were returned as a JSON string in the body of the response
    * - ``404 NOT FOUND``
      - The entity, program, or run for which properties are being requested was not found
+
+**Note**: The metadata aggregation of program, application, stream and dataset associated with a program is
+deprecated and program runs are treated like any other entity to which metadata can be associated directly.
+To retrieve the direct metadata of program run please refer to
+:ref:`Metadata Properties <http-restful-api-metadata-properties>` and
+:ref:`Metadata Tags <http-restful-api-metadata-tags>`. An additional query parameter ``aggregateRun`` must be set to
+``false`` while operating with metadata directly associated with program runs.
+
+Metadata for Custom Entities
+============================
+
+Metadata can also be associated with custom entities. In CDAP entities are separated into two kinds:
+
+- CDAP Entities: These entities are entities which are defined by CDAP and have special meaning in CDAP. ``Namespace``,``Stream``, ``Application``, ``Dataset`` etc. are example of CDAP Entities.
```

Comment:
is this a name we use elsewhere? It's kind of confusing, because it implies custom entities are not 'cdap' entities. Standard Entities? Known Entities?

Reply:
These entities are entities which are ... -> These are system defined entities that have special meaning in CDAP.

Reply:
> is this a name we use elsewhere? It's kind of confusing, because it implies custom entities are not 'cdap' entities. Standard Entities? Known Entities?

We have used it in under 5 places i guess. We can change it if we want. Are you suggesting to use Standard Entities for everything and Known for CDAP Entities.

Reply:
>These entities are entities which are ... -> These are system defined entities that have special meaning in CDAP.

Done.

Reply:
I think Custom Entities is fine, but suggesting that we change CDAP Entities to Standard Entites or Known Entities or some other name.